patches/components-permissions-request_type.cc.patch

```diff
@@ -1,16 +1,24 @@
 diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
-index 7e70c3b195ed2daa118dab140e6323e94b81e8d8..4a0e4b4673a05c99a11cbbd06795817e53574229 100644
+index 7e70c3b195ed2daa118dab140e6323e94b81e8d8..1d93e20a14e55134551054a63d10592edc50cb2f 100644
 --- a/components/permissions/request_type.cc
 +++ b/components/permissions/request_type.cc
-@@ -105,6 +105,7 @@ const gfx::VectorIcon& GetIconIdDesktop(RequestType type) {
+@@ -54,6 +54,7 @@ int GetIconIdAndroid(RequestType type) {
+       return IDR_ANDROID_INFOBAR_PROTECTED_MEDIA_IDENTIFIER;
+     case RequestType::kStorageAccess:
+       return IDR_ANDROID_INFOBAR_PERMISSION_COOKIE;
++    BRAVE_GET_ICON_ID_ANDROID
+   }
+   NOTREACHED();
+   return 0;
+@@ -105,6 +106,7 @@ const gfx::VectorIcon& GetIconIdDesktop(RequestType type) {
        return vector_icons::kCookieIcon;
      case RequestType::kWindowPlacement:
        return vector_icons::kSelectWindowIcon;
 +    BRAVE_GET_ICON_ID_DESKTOP
```

Comment:
same as BRAVE_GET_ICON_ID_ANDROID

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/63bceae1e84192ea16a32bd815f4fe5674f8f6b3